decidim-budgets/app/forms/decidim/budgets/admin/project_import_proposals_form.rb

```diff
@@ -9,12 +9,15 @@ class ProjectImportProposalsForm < Decidim::Form
         mimic :proposals_import
 
         attribute :origin_component_id, Integer
+        attribute :decidim_scope_id, Integer
```

Comment:
Why do we need to create two attributes, decidim_scope_id and scope_id into this Form ?
As below, in spec, we use scope_id attribute in form params and also in new.html.erb:
![image](https://user-images.githubusercontent.com/9702463/95184379-91be8280-07c7-11eb-9f11-f87734619754.png)